### DISCUSSION TITLE
Manually-specified size and position

### DISCUSSION THREAD

**Discussion by dabrahams** [rxhanson/Rectangle#678] [Q&A]
I realize this is a niche use-case, and it's the sort of thing I'd be happy to do from the terminal if there was a way: I often need to put a Safari window of a particular size at a particular place on the screen.  None of the preset sizes/positions will do.  Is there a backdoor I could use for this?

Thanks!

**Comment by rxhanson** [comment]
Not in Rectangle, but it is in my paid superset of Rectangle, Hookshot, in the custom shortcut builder. https://hookshot.app

**Reply by leggett** [reply]
Didn't even know about Hookshot -- happy to pay for this. Grateful to finally be able to resize to a specific resolution. Small suggestion: maybe add a promo inside Rectangle that says additional functionality can be had by upgrading to Hookshot?

**Reply by rxhanson** [reply]
Thanks for the suggestion! At a start, I’m going to change up Rectangle’s site, and will think through how to point users to Hookshot for certain desired features.
